ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -159,6 +164,7 @@ public void shouldInstantRegisterTopic() {
     // Given:
     final QueuedCommandStatus cmdStatus
         = new QueuedCommandStatus(new CommandId("TABLE", "orders", "CREATE"));
+    cmdStatus.setCommandOffset(0);
```

Comment:
meh, this isn't pertinent to this test. This muddies the water on what this test is testing.

However, I think this will all go way if you split `QueuedCommandStore` as I've suggested.